### PR TITLE
Implemented languages step for stepper with adaptivity

### PR DESCRIPTION
**Pull request by Yurenko**
Implemented languages step for stepper with adaptivity
image
image
image

### CODE REVIEW

File: src/components/step-wrapper/StepWrapper.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const StepWrapper = ({ children, steps }) => {
       <Box sx={ styles.steps }>
         { stepLabels }
       </Box>
-      <Box mt='30px'>
+      <Box mt='30px' sx={ { display: { xs: 'flex', sm: 'block' }, flex: 1 } }>
```
Comment (MickhaelSh) [inline src/components/step-wrapper/StepWrapper.js:66]: move styles to styles file

File: src/containers/tutor-home-page/language/Language.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  const { t } = useTranslation()
+  const { isDesktop, isMobile } = useBreakpoints()
+  const { stepData, handleStepData } = useStepContext()
+  const languageData = stepData[stepLabel]
+
+  const onChangeLanguage = (_, value) => {
+    if (value) {
+      handleStepData(stepLabel, value)
+    }
+    handleStepData(stepLabel, '')
```
Comment (Tolik170) [inline src/containers/tutor-home-page/language/Language.js:22]: do you need this else statement?

File: src/constants/translations/ua/become-tutor.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "stepLabels": {
+    "generalInfo": "Загальне", 
+    "language": "Мова", 
+    "subjects": "Предмети",
+    "photo": "Фото"
+  },
+  "generalInfo": {
+    "title": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint.",
+    "textFieldLabel": "Коротко опишіть свій професійний статус",
+    "checkboxLabel": "Я підтверджую, що мені більше 18 років",
```
Comment (Tolik170) [inline src/constants/translations/ua/become-tutor.json:11]: As i remember we didn`t use chekbox for age anymore, i forget to remove it, can you?

File: src/components/step-wrapper/StepWrapper.js (hunk 1 above)
Comment (Yurenko) [reply src/components/step-wrapper/StepWrapper.js]: Fixed

File: src/constants/translations/ua/become-tutor.json (hunk 3 above)
Comment (Yurenko) [reply src/constants/translations/ua/become-tutor.json]: I did it)

File: src/containers/tutor-home-page/language/Language.js (hunk 2 above)
Comment (Yurenko) [reply src/containers/tutor-home-page/language/Language.js]: It doesn't need here. I fixed it.

**Review by Tolik170 — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Pull request merged**